recipe/meta.yaml

```diff
@@ -11,29 +11,52 @@ source:
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install ./package -vv
-  number: 0
+  script:
```

Comment:
As far as I understand, a `.sh` build script can be provided separately, see for example

https://github.com/conda-forge/fenics-feedstock/blob/12ef077802cc9fad34cf984ec7af80585b44301b/recipe/meta.yaml#L180-L186

Reply:
So, these instructions could be a `build.sh`

Reply:
Yes - for sure !